From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Sami Tolvanen <samitolvanen@google.com>
Date: Thu, 21 Nov 2019 14:22:31 -0800
Subject: ANDROID: gki_defconfig: disable FUNCTION_TRACER

Remove CONFIG_FUNCTION_TRACER as it's not needed. Note that
CONFIG_FTRACE is still enabled by default.

Change-Id: Id56ce42429360b59d70c2679db5ea5ff439544a0
Signed-off-by: Sami Tolvanen <samitolvanen@google.com>
---
 arch/arm64/configs/gki_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index 4c4a83cb4e76..70b13d6ebe93 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -479,7 +479,6 @@ CONFIG_SOFTLOCKUP_DETECTOR=y
 # CONFIG_DETECT_HUNG_TASK is not set
 CONFIG_PANIC_TIMEOUT=5
 CONFIG_SCHEDSTATS=y
-CONFIG_FUNCTION_TRACER=y
 # CONFIG_RUNTIME_TESTING_MENU is not set
 CONFIG_CORESIGHT=y
 CONFIG_CORESIGHT_STM=y
